scrapy_zyte_api/addon.py

```diff
@@ -0,0 +1,24 @@
+from scrapy.settings import BaseSettings
+
+from scrapy_zyte_api import ScrapyZyteAPIDownloaderMiddleware
+
+
+class ScrapyZyteAPIAddon:
+    def update_settings(self, settings: BaseSettings) -> None:
```

Comment:
I wonder if addon should set ZYTE_API_TRANSPARENT_MODE=True by default, as it's a recommended way to use the package.

Reply:
In this case, we still should respect ZYTE_API_TRANSPARENT_MODE=False setting set by the user. 

Probably, we'd need a different default value for the option, not False, to be able to distinguish between these two cases, but maybe there is another way to solve it.

Reply:
Why do we need to distinguish them?

Reply:
If user sets ZYTE_API_TRANSPARENT_MODE=False in their settings, we should respect that. I guess you can handle it by using settings priority directly, instead of having True/False/DEFAULT.

Reply:
I think the code I added (which uses priorities) is enough for this, and we have a test for that.